.github/uffizzi/docker-compose.uffizzi.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+
+x-uffizzi:
+  ingress:
+    service: backstage
+    port: 7007
+
+services:
+  backstage:
+    image: '${BACKSTAGE_IMAGE}'
+    environment:
+      POSTGRES_HOST: localhost
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: example
+      GITHUB_TOKEN: abc
+      NODE_ENV: production
+    restart: unless-stopped
+    ports:
+      - '7007:7007'
+    deploy:
+      resources:
+        limits:
+          memory: 500M
+    entrypoint: "/bin/sh"
+    command:
+      - "-c"
+      - "APP_CONFIG_app_baseUrl=$$UFFIZZI_URL APP_CONFIG_backend_baseUrl=$$UFFIZZI_URL APP_CONFIG_auth_environment='production' node packages/backend --config app-config.yaml --config app-config.production.yaml"
+
+  adminer:
```

Comment:
I don't see any way to connect to this container? Should we remove it or add an `nginx` proxy container?

Reply:
I think we can remove this container.